packages/lightning-lsp-common/src/context.ts

```diff
@@ -39,23 +37,6 @@ async function findSubdirectories(dir: string): Promise<string[]> {
     return subdirs;
 }
 
-/**
- * @return list of .js modules inside namespaceRoot folder
- */
-async function findModulesIn(namespaceRoot: string): Promise<string[]> {
-    const files: string[] = [];
-    const subdirs = await findSubdirectories(namespaceRoot);
-    for (const subdir of subdirs) {
-        const basename = path.basename(subdir);
-        const modulePath = path.join(subdir, basename + '.js');
-        if ((await fs.pathExists(modulePath)) && componentUtil.isJSComponent(modulePath)) {
```

Comment:
removing this made componentUtil unused import.

can you adjust the lint/ts rules to prevent that kind of thing?